.circleci/config.yml

```diff
@@ -951,9 +951,7 @@ workflows:
             branches:
               only:
                 - irs
-                - migration
```

Comment:
don't want to perform smoketests in `migration` or `test` environment that would contaminate migration data

Reply:
Partial revert of https://github.com/ustaxcourt/ef-cms/pull/482, linking in for future reference.